packages/components/src/core/Autocomplete/index.tsx

```diff
@@ -1,309 +1,239 @@
+  T,
+  Multiple extends boolean | undefined,
+  DisableClearable extends boolean | undefined,
+  FreeSolo extends boolean | undefined
+> =
+  | AutocompleteSingleColumnValue<T, Multiple, DisableClearable, FreeSolo>
+  | AutocompleteMultiColumnValue<T, Multiple, DisableClearable, FreeSolo>;
+
+export type AutocompleteSingleColumnOnChange<
+  T,
+  Multiple,
+  DisableClearable,
+  FreeSolo
+> = (
+  event: React.SyntheticEvent,
+  value: AutocompleteValue<T, Multiple, DisableClearable, FreeSolo>,
+  reason: AutocompleteChangeReason,
+  details?: AutocompleteChangeDetails<T>
+) => void;
+
+export type AutocompleteMultiColumnOnChange<
+  T,
+  Multiple extends boolean | undefined,
+  DisableClearable extends boolean | undefined,
+  FreeSolo extends boolean | undefined
+> = (
+  event: React.SyntheticEvent,
+  value: AutocompleteMultiColumnValue<T, Multiple, DisableClearable, FreeSolo>,
+  reason: AutocompleteChangeReason,
+  details?: AutocompleteChangeDetails<T>
+) => void;
+
+export type SDSAutocompleteOnChange<
+  T,
+  Multiple extends boolean | undefined,
+  DisableClearable extends boolean | undefined,
+  FreeSolo extends boolean | undefined
+> =
+  | AutocompleteSingleColumnOnChange<T, Multiple, DisableClearable, FreeSolo>
+  | AutocompleteMultiColumnOnChange<T, Multiple, DisableClearable, FreeSolo>;
+
+interface ExtraAutocompleteProps<
+  T,
+  Multiple extends boolean | undefined,
+  DisableClearable extends boolean | undefined,
+  FreeSolo extends boolean | undefined
+> {
+  options: SDSAutocompleteOptions<T, Multiple, DisableClearable, FreeSolo>;
+  PopperBaseProps?: Partial<PopperProps>;
+  value?: SDSAutocompleteValue<T, Multiple, DisableClearable, FreeSolo>;
+  onClickAway?: (event: MouseEvent | TouchEvent) => void;
+  onChange?: SDSAutocompleteOnChange<T, Multiple, DisableClearable, FreeSolo>;
+  count?: number;
+  icon?: ReactElement;
+  search?: boolean;
 }
 
-type CustomAutocompleteProps<
+export type AutocompleteProps<
   T,
-  Multiple extends boolean | undefined = undefined,
-  DisableClearable extends boolean | undefined = undefined,
-  FreeSolo extends boolean | undefined = undefined
+  Multiple extends boolean | undefined,
+  DisableClearable extends boolean | undefined,
+  FreeSolo extends boolean | undefined
 > = Omit<
-  MuiAutocompleteProps<T, Multiple, DisableClearable, FreeSolo>,
-  "renderInput" | "nonce" | "rev" | "rel" | "autoFocus" | "content"
->;
+  AutocompleteBaseProps<T, Multiple, DisableClearable, FreeSolo>,
+  "options" | "value" | "onChange"
+> &
+  ExtraAutocompleteProps<T, Multiple, DisableClearable, FreeSolo>;
 
-export type AutocompleteProps<
-  T,
-  Multiple extends boolean | undefined = undefined,
-  DisableClearable extends boolean | undefined = undefined,
-  FreeSolo extends boolean | undefined = undefined
-> = CustomAutocompleteProps<T, Multiple, DisableClearable, FreeSolo> &
-  ExtraAutocompleteProps;
+export { StyledPopper as AutocompleteMultiColumnStyledPopper };
```

Comment:
Thanks for exporting this!

Can we add a comment that for single column case, people can just import Popper from MUI instead of using `AutocompleteMultiColumnStyledPopper`? (At least that's my impression 😆 )

Thank you!

Reply:
Thats right. I'll add a comment in there :D